### ISSUE TITLE
admesh hang on the input

### ISSUE THREAD

**Issue by tianxiaogu** [admesh/admesh#29]
admesh hangs on the attached input.

[h0.zip](https://github.com/admesh/admesh/files/2373130/h0.zip)

```
./admesh h0
```

**Comment by hroncok** [comment]
Thanks for the report. I unfortunately won't have time to look at it any time soon.

**Comment by udif** [comment]
(Yes I see this is a 2.5 year bug report).

I took a look at the attached file, and it is a corrupted STL ASCII file.
So while admesh could be fixed to be more strict reporting errors, the file is clearly in error, just open it in any text editor.

**Comment by hroncok** [comment]
Note that hanging on invalid/crafted input is usually considered a security vulnerability because it can lead to denial of service.

**Comment by udif** [comment]
You are right. Looking at https://github.com/admesh/admesh/issues/28 I realized where the corrupted files came from. I wasn't aware of [ASAN](https://github.com/google/sanitizers/wiki/AddressSanitizer), so maybe I'll take a look and learn something from it.
